application/event_processor/tests/test_dos.py

```diff
@@ -410,3 +420,7 @@ def test_get_dos_locations(mock_connect):
     del environ["DB_SCHEMA"]
     del environ["DB_USER_NAME"]
     del environ["DB_PASSWORD"]
+
+
+def test_disconnect_dos_db():
```

Comment:
Can this test function assert that a connection is closed. Maybe use magicmock as the db connection and assert the close method is called.